Actors/VisualObjects/VisualObjects.WebService/wwwroot/Index.html

```diff
@@ -1,4 +1,5 @@
-﻿<html>
+﻿<!DOCTYPE html>
```

Comment:
Setting the doctype actually messes with the rendering of the box, making it super small. You can tell from your screenshot that you had to zoom way in to see the display which shouldn't be the case; it should take up the whole browser window.